charts/openshift-metering/templates/hive/hive-configmap.yaml

```diff
@@ -31,46 +31,52 @@ data:
       </property>
       <property>
         <name>hive.metastore.uris</name>
-        <value>{{ .Values.presto.spec.hive.config.metastoreURIs }}</value>
+        <value>{{ .Values.hive.spec.config.metastoreURIs }}</value>
       </property>
```

Comment:
I don't see in hive.spec.config the metastoreURIS field

Reply:
oh nvm... see it!

Reply:
https://github.com/operator-framework/operator-metering/pull/783/files#diff-60c0d87b6c8a5d942ed6b466067f8dfdR574